dev/releases/release_notes.py

```diff
@@ -316,6 +334,27 @@ def changes_overview(
         # finally copy over this new file to changelog.md
         os.rename(newfile, finalfile)
 
+def split_pr_into_changelog(prs: List):
+    childprlist = []
+    for pr in prs:
+        if has_label(pr, 'release notes: use body'):
+            mdstring = body_to_release_notes(pr).strip()
+            mdlines = mdstring.split('\r\n')
+            pattern = r'\{package: .*\}'
+            for line in mdlines:
+                if not '{package: ' in line:
+                    continue
+                mans = re.search(pattern, line)
```

Comment:
Instead of doing a partial check, then `re.search`, and running into an exception if the regex did not match, you should inspect the return value of `re.search` to decide if there was no match.

(And if there was no match, *and* the input is non empty, we should indicate an error)